common/server_room.cpp

```diff
@@ -232,6 +234,22 @@ void Server_Room::say(const QString &userName, const QString &s, bool sendToIsl)
     event.set_name(userName.toStdString());
     event.set_message(s.toStdString());
     sendRoomEvent(prepareRoomEvent(event), sendToIsl);
+ 
+    if (chatHistorySize != 0) {
+
+        ServerInfo_ChatMessage chatMessage;
+        QDateTime dateTime = dateTime.currentDateTime();
+        QString dateTimeString = dateTime.toString("MM/dd/yy hh:mm:ss");
```

Comment:
I actually had it set to show the originally formated QDateTime currentDateTime output.  But was asked by @ZeldaZach to change it to the format that you see now.  I don't really care myself.  I can change it to whatever.  What would everyone like?